traitlets/config/tests/test_application.py

```diff
@@ -179,6 +186,7 @@ class TestApp(Application):
             assert app.config.TestApp.value == 'cli'
             assert app.value == 'cli'
 
+    @skip('TO DEL? see https://github.com/ipython/traitlets/pull/322')
```

Comment:
We need to make sure this test still passes. If it doesn't, then something is broken that needs to be fixed.